docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -44,7 +44,9 @@ See [Troubleshooting](doc:troubleshooting#how-to-change-your-cache-directory) fo
 >    size_mb=$(du -m -d0 "${path}" | cut -f 1)
 >    if (( size_mb > limit_mb )); then
 >      echo "${path} is too large (${size_mb}mb), nuking it."
->      rm -rf "${path}"
+>      nuke_path="${path}.$(uuidgen)"
+>      mv "${path}" "${nuke_path}"
+>      rm -rf "${nuke_path}"
```

Comment:
This is frustrating. FWICT uuidgen is not widely available - you do add a note above - but people just copy / paste (as you know!). Maybe `mkdtemp` which is widely available and can make a non-colliding dir:
```
$ docker run --rm -it ubuntu:22.04 which uuidgen || docker run --rm -it ubuntu:22.04 which mktemp
/usr/bin/mktemp
```

Also `rm -rf "${path}.*"` seems better since it will clean up the cruft that accumulates across multiple cancelled jobs.

Reply:
Good idea. Done.